src/app/_components/NotFound404.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Link from 'next/link';
+import NotFound from '@images/notFound.svg';
+import PropTypes from 'prop-types';
+
+export function NotFound404({ children, href = '/', hrefText = 'Повернутися на головну', className }) {
```

Comment:
Rename to NotFoundLayout or NotFoundPage

Reply:
Renamed